src/shared/tar-util.c

```diff
@@ -113,14 +115,22 @@ static int open_inode_apply_acl(OpenInode *of) {
                 return 0;
 
         if (of->acl_access) {
+#if HAVE_ACL
                 if (sym_acl_set_fd(of->fd, of->acl_access) < 0)
                         RET_GATHER(r, log_error_errno(errno, "Failed to adjust ACLs of '%s': %m", of->path));
+#else
+                log_debug("The archive entry '%s' has ACLs, but ACL support is disabled.", of->path);
```

Comment:
i'd prefer if this would say that we ignore the ACL data. i.e. just end this in ", not applying ACL."